Error while using configuration wizard

While setting up a node (version `3.2.0`) with the configuration wizard I encountered the an error just after filling in the path to the private key file.

I don't know what to make of the error and it seems pretty generic. Could there be a bug in the configuration wizard?

```bash
? Path to private key file: /home/some/path/private.pem
Traceback (most recent call last):
  File "/home/some/path/venv/bin/vnode", line 8, in <module>
    sys.exit(cli_node())
  File "/home/some/path/venv/lib/python3.8/site-packages/click/core.py", line 1128, in __call__
    return self.main(*args, **kwargs)
  File "/home/some/path/venv/lib/python3.8/site-packages/click/core.py", line 1053, in main 
    rv = self.invoke(ctx)
  File "/home/some/path/venv/lib/python3.8/site-packages/click/core.py", line 1659, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/some/path/venv/lib/python3.8/site-packages/click/core.py", line 1395, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/some/path/venv/lib/python3.8/site-packages/click/core.py", line 754, in invoke
    return __callback(*args, **kwargs)
  File "/home/some/path/venv/lib/python3.8/site-packages/vantage6/cli/node.py", line 171, in cli_node_new_configuration
    cfg_file = configuration_wizard("node", name, environment, system_folders)
  File "/home/some/path/venv/lib/python3.8/site-packages/vantage6/cli/configuration_wizard.py", line 241, in configuration_wizard
    config_manager.put(environment, config)
  File "/home/some/path/venv/lib/python3.8/site-packages/vantage6/common/configuration_manager.py", line 73, in put
    configuration = self.conf_class(config)
  File "/home/some/path/venv/lib/python3.8/site-packages/vantage6/common/configuration_manager.py", line 14, in __init__
    super().__init__(*args, **kwargs)
  File "/usr/lib/python3.8/collections/__init__.py", line 999, in __init__
    self.update(dict)
  File "/usr/lib/python3.8/_collections_abc.py", line 832, in update
    self[key] = other[key]
  File "/home/some/path/venv/lib/python3.8/site-packages/vantage6/common/configuration_manager.py", line 24, in __setitem__
    assert schema.is_valid(value), f"Invalid Value! {value} for {schema}"
AssertionError: Invalid Value!  for Schema(And(Use(<class 'str'>), <built-in function len>))
```

--- a/vantage6-common/vantage6/common/configuration_manager.py
+++ b/vantage6-common/vantage6/common/configuration_manager.py
@@ -21,7 +21,8 @@ class Configuration(collections.UserDict):
             self.VALIDATORS.get(key, lambda x: True),
             ignore_extra_keys=True
         )
-        assert schema.is_valid(value), f"Invalid Value! {value} for {schema}"
+        assert schema.is_valid(value), \
+            f"Invalid value '{value}' provided for field '{key}'"
         super().__setitem__(key, value)
 
     def __getitem__(self, key):

--- a/vantage6-server/vantage6/server/configuration/configuration_manager.py
+++ b/vantage6-server/vantage6/server/configuration/configuration_manager.py
@@ -9,6 +9,8 @@ from vantage6.common.configuration_manager import (
 )
 
 
+# FIXME BvB 2022-08-10: this code is duplication of vantage6/cli/config_manager
+# Remove in one of both places
 class ServerConfiguration(Configuration):
 
     VALIDATORS = {

--- a/vantage6/vantage6/cli/configuration_manager.py
+++ b/vantage6/vantage6/cli/configuration_manager.py
@@ -54,24 +54,6 @@ class NodeConfiguration(Configuration):
     }
 
 
-class TestConfiguration(Configuration):
-
-    VALIDATORS = {
-        # "api_key": And(Use(str), len),
-        # "logging": {
-        #     "level": And(Use(str), lambda l: l in ("DEBUG", "INFO",
-        #                                            "WARNING", "ERROR",
-        #                                            "CRITICAL", "NONE")),
-        #     "file": Use(str),
-        #     "use_console": Use(bool),
-        #     "backup_count": And(Use(int), lambda n: n > 0),
-        #     "max_size": And(Use(int), lambda b: b > 16),
-        #     "format": Use(str),
-        #     "datefmt": Use(str)
-        # }
-    }
-
-
 class NodeConfigurationManager(ConfigurationManager):
 
     def __init__(self, name, *args, **kwargs):

--- a/vantage6/vantage6/cli/configuration_manager.py
+++ b/vantage6/vantage6/cli/configuration_manager.py
@@ -31,7 +31,7 @@ class ServerConfiguration(Configuration):
 class NodeConfiguration(Configuration):
 
     VALIDATORS = {
-        "api_key": And(Use(str), len),
+        "api_key": Use(str),
         "server_url": Use(str),
         "port": Or(Use(int), None),
         "task_dir": Use(str),

